gradunwarp/core/tests/data/gradunwarp_coeffs.grad

```diff
@@ -0,0 +1,65 @@
+#*[ Script ****************************************************************\
+#
+# Name        : dummy.grad
+#
+# Date        : 0970-01-01
+#
+# Author      : Mario Bros
+#
+# Language    :
+#
+# Description : Defines Legendre coefficients in spherical harmonics for
+#               dummy Gradient Coil 
+#
+#****************************************************************************/
+
+#*] END: */
+dummy, dummy , Gx,y,z = 80/80 mT/m
+win_low = 0, win_high = 0, win_algo = 0, win_dummy = 2;
+0.25 m = R0, lnorm = 4? A(1,0) = B(1,1) = A(1,1) = 0;
+0 = CoSyMode,
+NO.    TYPE                  SPECTRUM                                AXIS
+
+1 A( 3, 0)           -0.0510000                  z
```

Comment:
Were these all replaced with random values?  Sounds like we should get sign-off from the vendor that you used as a starting point for this dummy file...